_includes/head.html

```diff
@@ -2,7 +2,7 @@
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="shortcut icon" type="image/png" href="imgs/favicon.png"/>
+  <link rel="shortcut icon" type="image/png" href="assets/imgs/favicon.png"/>
```

Comment:
Depending on which page will be shown this relative path will be wrong. I checked how they do it with the other assets and it looks like they have a solution for that: `<link rel="stylesheet" href="{{ "/assets/main.css" | relative_url }}">`  so if you replace `href="assets/imgs/favicon.png"` with `href="{{ "/assets/imgs/favicon.png" | relative_url }}">` it should work fine everywhere.